core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -124,7 +124,9 @@ class HttpGoogleServicesDAO(
                              cleanupDeploymentAfterCreating: Boolean,
                              terraBucketReaderRole: String,
                              terraBucketWriterRole: String,
-                             override val accessContextManagerDAO: AccessContextManagerDAO)(implicit val system: ActorSystem, val materializer: Materializer, implicit val executionContext: ExecutionContext, implicit val cs: ContextShift[IO], implicit val timer: Timer[IO]) extends GoogleServicesDAO(groupsPrefix) with FutureSupport with GoogleUtilities {
+                             override val accessContextManagerDAO: AccessContextManagerDAO,
+                             resourceBufferPemFile: String,
```

Comment:
we should use the json instead of pem. That makes the firecloud-develop stuff simpler. Use `GoogleCredential.fromStream(...).toBuilder.setServiceAccountScopes(...).build()`